scripts/cloudbuild.sh

```diff
@@ -43,5 +44,8 @@ then
   curl -s https://raw.githubusercontent.com/GoogleCloudPlatform/python-runtime/master/scripts/local_cloudbuild.py | \
   python3 - --config=$projectRoot/target/cloudbuild_local.yaml --output_script=$projectRoot/target/cloudbuild_local.sh
 else
+  # invoke cloud container builder to build the image
   gcloud container builds submit --config=$projectRoot/target/cloudbuild.yaml .
+  # run integration tests on the built image
+  source $dir/integration_test.sh $IMAGE
```

Comment:
I actually think it might just make more sense to not call the integration_test script directly from the cloudbuild script. I think that's better than implementing a --skip-integration-test arg to the cloudbuild script. 

I'll also modify integration_test.sh to accept a `--local` arg.